extensions/smallrye-jwt-build/deployment/src/main/java/io/quarkus/smallrye/jwt/builder/deployment/SmallRyeJwtBuildProcessor.java

```diff
@@ -3,6 +3,7 @@
 import io.quarkus.deployment.annotations.BuildProducer;
```

Comment:
Hmmm. The extension is called `smallrye-jwt-build` but the root package is called `smallrye.jwt.builder`. It should be consistent.

It's not related to this PR but we need to fix the root packages of both the deployment module and the runtime module.

Reply:
@gsmet Hi Guillaume - I referred to it though indirectly when I was adding `quarkus-smallrye-jwt` - the reason behind it is that `.build.` is in the list of the ignored packages in Quarkus - I think it may be a legacy so if removing this restriction is fine with you then I'll update the deployment package - note the subpackage name of the JWT is also `.build.`.  I think going forward I may need to rename `.build.` to `.builder.` everywhere. I propose to discuss it further on `dev`